CHANGELOG.md

```diff
@@ -72,6 +72,9 @@ These changes are available on the `master` branch, but have not yet been releas
 - Added `suppress` and `allowed_mentions` parameters to `Webhook` and
   `InteractionResponse` edit methods.
   ([#2138](https://github.com/Pycord-Development/pycord/pull/2138))
+- Added `Guild.create_forum_channel` parameter and `ForumChannel` attribute:
+  `default_reaction_emoji`.
```

Comment:
```suggestion
- Added `default_reaction_emoji` parameter to `Guild.create_forum_channel()`, `ForumChannel.create_forum_channel()`, and
`ForumChannel.edit_forum_channel()` functions:
```

Reply:
Probably also add a separate changelog entry about the `default_reaction_emoji` attribute.